### PR TITLE
preserve the relative path and filename of JES outputs within the bucket

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala [hunk 1]
```diff
@@ -127,7 +127,7 @@ class JesBackend extends Backend with LazyLogging {
       val evaluatedExpression = taskOutput.expression.evaluate(scopedLookupFunction, engineFunctions, interpolateStrings = true)
       evaluatedExpression match {
         case Success(v) =>
-          val jesOutput = JesOutput(taskOutput.name, s"$callGcsPath/${taskOutput.name}", Paths.get(v.valueString))
+          val jesOutput = JesOutput(taskOutput.name, s"$callGcsPath/${v.valueString}", Paths.get(v.valueString))
```
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: I'm not actually sure what this change fixes or why it was made... can you explain?

Comment (mcovarr) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: ha glad it's not just me.  :smile:

**Comment by scottfrazer** [comment]
@mcovarr next

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala (hunk 1 above)
Comment (geoffjentry) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: Discussed in person but for posterity call outputs are written to the call's directory and the path is retained so two files with the same name won't conflict

Comment (mcovarr) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: :+1:

**Pull request merged**